parser-typechecker/src/Unison/Codebase/Causal.hs

```diff
@@ -132,27 +164,38 @@ instance Hashable (RawHash h) where
 -- Find the lowest common ancestor of two causals.
 lca :: Monad m => Causal m h e -> Causal m h e -> m (Maybe (Causal m h e))
 lca a b =
-  go Set.empty Set.empty (Seq.singleton $ pure a) . Seq.singleton $ pure b
- where
+  lca' (Seq.singleton $ pure a) . Seq.singleton $ pure b
```

Comment:
I know this has been in here a while now, but as long as we're modifying it, could we flip it around to make the arguments equal?
```haskell
lca' (Seq.singleton $ pure a) (Seq.singleton $ pure b)
```